### DISCUSSION TITLE
Azure Cost (noob question)

### DISCUSSION THREAD

**Discussion by dantedantas** [AlexaCRM/dynamics-webapi-toolkit#68] [Q&A] [answered]
Hello,

I want to registrate my App. Will I have any kind of costs?

![image](https://user-images.githubusercontent.com/10854571/126653180-bff06984-1839-494a-9ad0-21ffea0ad35a.png)

Thank you!

BR
Dante

**Comment by georged** [comment, marked as answer]
No, it's free.
For Azure charges please check https://azure.microsoft.com/pricing/

**Comment by dantedantas** [comment]
Thank you Georg.
